nestle.py

```diff
@@ -583,6 +651,13 @@ def update(self, pointvol):
         self.ell = bounding_ellipsoid(self.points, pointvol=pointvol)
         self.ell.scale_to_vol(pointvol)
 
+    def propose_point(self, u):
+        while True:
+            new_u = u + scale * self.ell.randoffset(rstate=self.rstate)
```

Comment:
> Not as far as I can tell, since this particular propose_point (and its use of scale) is already specific to the MCMC implementation.

But the `Sampler` base class calls `propose_point()` in `fill_queue()`, so the interface has to be the same for all derived subclasses.